### PR TITLE
py-setuptools_scm_git_archive: new port

### PR DESCRIPTION
**Pull request by GiovanniBussi**
###### Type(s)

- [ ] bugfix
- [ ] enhancement
- [ ] security fix

###### Tested on

macOS 10.11.6 15G22010
Xcode 8.1 8B62 

###### Verification 
Have you

- [X] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [X] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [X] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?

- [X] checked your Portfile with `port lint`?
- [ ] tried existing tests with `sudo port test`?
- [X] tried a full install with `sudo port -vst install`?
- [X] tested basic functionality of all binary files?

### CODE REVIEW

File: python/py-setuptools_scm_git_archive/Portfile [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+version             1.0
+categories-append   devel
+platforms           darwin
+license             MIT
+supported_archs     noarch
+
+master_sites        pypi:s/setuptools_scm_git_archive
+distname            setuptools_scm_git_archive-${version}
+
+python.versions     27 34 35 36 37
```
Comment (pmetzger) [inline python/py-setuptools_scm_git_archive/Portfile]: New ports should not add 3.4 or 3.5, both are obsolete.

File: python/py-setuptools_scm_git_archive/Portfile [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+long_description    ${description}
+
+homepage            https://github.com/Changaco/setuptools_scm_git_archive
+
+checksums           rmd160  665c374d66941d878284e72dcd40125fb88debd3 \
+                    sha256  52425f905518247c685fc64c5fdba6e1e74443c8562e141c8de56059be0e31da \
+                    size    3050
+
+if {${name} ne ${subport}} {
+    depends_lib-append  port:py${python.version}-setuptools_scm
```
Comment (pmetzger) [inline python/py-setuptools_scm_git_archive/Portfile]: We seem to be missing the usual bit that turns off livecheck off for the subports. (I wonder if that shouldn't just be part of the portgroup?)

Comment (GiovanniBussi) [reply python/py-setuptools_scm_git_archive/Portfile]: Sorry my fault... I added it back

File: python/py-setuptools_scm_git_archive/Portfile (hunk 1 above)
Comment (GiovanniBussi) [reply python/py-setuptools_scm_git_archive/Portfile]: Fixed in the new commit

**Comment by GiovanniBussi** [comment]
@pmetzger I think I fixed both problems (sorry maybe I am answering twice)

**Comment by pmetzger** [comment]
Merged. Thanks, @GiovanniBussi!

**Pull request merged**